packages/eslint-plugin-millionjs/docs/rules/avoid-non-deterministic-returns.md

```diff
@@ -0,0 +1,47 @@
+# This ensures that returns must be deterministic. There can only be one return statement at the end of the block that returns a stable tree
+
+Please describe the origin of the rule here.
+
+## Rule Details
+
+This rule aims to...
+
+Examples of **incorrect** code for this rule:
+
+```js
+function Component() {
+  const [count, setCount] = useState(initial.count);
+ 
+  if (count > 10) {
+    return <div>Too many clicks!</div>; // ❌ Wrong
+  }
+ 
+  // ❌ Wrong
+  return count > 5 ? (
+    'Count is greater than 5'
+  ) : (
+    <div>Count is {count}.</div>
+  );
+}
+ 
+const ComponentBlock = block(Component);
+```
+
+Examples of **correct** code for this rule:
+
+````js
+function Component() {
+  const [count, setCount] = useState(initial.count);
+ 
+ 
+  return count > 5 ? (
+    'Count is greater than 5'
+  ) 
```

Comment:
Something is wrong here

Reply:
as how
could you please elaborate
I would really appreciate that

Reply:
ohhh
i see now
do you mean no closing for the ternary?

Reply:
Yep :)

Reply:
I probably just forgot ..😅😅 thanks